t/obs/openSUSE:Leap:15.4:Images:ToTest/print_openqa.before

```diff
@@ -1,4 +1,4 @@
-/usr/bin/openqa-cli api -X post isos \
+/usr/share/openqa/script/client isos post --host localhost \
  ARCH=x86_64 \
  ASSET_256=openSUSE-Leap-15.3-JeOS.x86_64-15.3-kvm-and-xen-Build4.9.qcow2.sha256 \
```

Comment:
@Vogtinator btw, this is suspicious... have we properly updated the version in JeOS images for 15.4?

Reply:
git sometimes behaves weird - not sure how the old command was used here. 
If I search in github - I cannot find any reference to `/usr/share/openqa/script/client` 
I'd just get new `git clone` of openqa-trigger-from-obs and retry this PR with it. Otherwise I can do the same and see if it gets correct this time.

Reply:
The JeOS images have not been updated yet - waiting for the Kiwi update to land in 15.4.